src/utils/uploader.js

```diff
@@ -129,11 +125,10 @@ class Uploader {
 
     // upload
     async upload(path, gasIncPct, threadPoolSize) {
-        const syncPoolSize = threadPoolSize || DEFAULT_THREAD_POOL_SIZE;
         const results = [];
         return new Promise((resolve, reject) => {
             from(recursiveFiles(path, ''))
-                .pipe(mergeMap(info => this.#upload(info, gasIncPct), syncPoolSize))
+                .pipe(mergeMap(info => this.#upload(info, gasIncPct), threadPoolSize))
```

Comment:
That is the reason that many teams choose typescript instead of javascript because TS can have type check for us